storybook/native/stories.js

```diff
@@ -4,8 +4,73 @@ import Button from '../../src/components/Button/native';
 import Text from '../../src/components/Text/native';
 
 storiesOf('Button', module)
```

Comment:
Add all variants of the Button like how the web story does

Reply:
Added them individually as we don't have an Aux or padding component yet on native. Once that comes in, will bunch the kinds and shapes together